src/main/resources/jupyter/google_plugin_jupyterlab.js

```diff
@@ -0,0 +1,7 @@
+module.exports = [{
+    id: 'google_plugin_jupyterlab',
+    autoStart: true,
+    activate: function(app) {
+        require('/home/jupyter-user/.jupyter/custom/google_sign_in');
```

Comment:
This is the entrypoint for JupyterLab extensions. In this case it just loads another file.

Note this has to be an absolute path, as for relative paths it defaults to looking for node packages in some other directory.